extern/region-4.14/ltmain.sh

```diff
@@ -64,7 +64,7 @@ package_revision=2.4.6
 # libraries, which are installed to $pkgauxdir.
 
 # Set a version string for this script.
-scriptversion=2015-10-12.13; # UTC
+scriptversion=2015-01-20.17; # UTC
```

Comment:
I'm going to note that this appears to be a reversion but it looks like this is due to `2.4.6 Debian 2.4.6-4` has changed to `2.4.6` so I'm not going to ask for this to be updated.